guide/deployment/deploying-to-docker.md

```diff
@@ -0,0 +1,154 @@
+
+## Manually building and publishing your Docker image
+
+Step 1: Build your Docker image
+
+```bash
+    docker build -t [your-image-name] .
+```
+
+Step 2: Push your Docker image to Docker Hub
+
+```bash
+    docker push [your-image-name]
+```
+
+## Automating the build process with GitHub Actions
+
+<Definition content="GitHub Actions is a CI/CD platform that allows you to automate your build, test, and deployment pipelines. You can set up workflows to build and test each pull request that comes into your repository, for this example we will be using GitHub Actions to publish a Docker Image every push you do to your master branch." />
+
+To use GitHub Actions your code must be in a GitHub repository.
+You can store your code in a git server such as GitHub for ease of access.
+
+### GitHub Desktop
+
+> A good choice for people who are scared of the terminal and don't wanna learn it, or simply just get things done quicker with a app.
+
+1. Download [GitHub Desktop](https://desktop.github.com/)
+2. [Create a repository](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/creating-your-first-repository-using-github-desktop)
+
+### Terminal
+
+> The classic way
+
+1. [Make sure git is installed](https://github.com/git-guides/install-git)
+2. [Create a repository](https://docs.github.com/en/get-started/quickstart/create-a-repo)
+
+### Creating the GitHub Action
+
+Make a new file called `publish.yml` in `.github/workflows` with the contents of:
+
+```yml
+# Name of the workflow
+name: publish
+
+# When to trigger it
+on:
+    # Trigger the workflow on push or pull request,
+    # but only for the main branch
+    push:
+        branches:
+            - main
+
+# Jobs to run
+jobs:
+    # Build the bot job,
+    build-bot:
+        # Run it on ubuntu
+        runs-on: ubuntu-latest
+
+        # Steps of the job
+        steps:
+            # Checkout the repo at the current state
+            - uses: actions/checkout@v2
+
+              # Login to the GitHub Container Registry
+            - name: Login to GitHub Container Registry
+              uses: docker/login-action@v1
+              with:
+                  registry: ghcr.io
+                  username: ${{ github.actor }}
+                  password: ${{ secrets.GITHUB_TOKEN }}
+
+              # Build and Push the bot to the GHCR
+            - name: Build and Push to GHCR
+              run: |
+                  docker build . --tag ghcr.io/AccountUsername/RepoName:latest
+                  docker push ghcr.io/AccountUsername/RepoName:latest
+```
+
+Now every time you push to your master branch the bot will be built and pushed to the GHCR.
+
+# Running Your Docker Container
+
+Now that we have containerized our bot, we can run it by typing:
```

Comment:
```suggestion
Now that you have containerized our bot, you can run it by typing:
```
https://github.com/discordjs/guide/blob/main/CONTRIBUTING.md#youyour-instead-of-weour